api/app/Models/CandidateSearchRequest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * Class Department
+ *
+ * @property int $id
+ * @property int $full_name
+ * @property array $email
+ * @property array $department_id
+ * @property array $job_title
+ * @property array $additional_comments
+ * @property array $summary_of_filters
+ * @property array $requested_date
+ * @property array $status
+ * @property array $admin_notes
+ * @property Illuminate\Support\Carbon $created_at
+ * @property Illuminate\Support\Carbon $updated_at
+ * @property Illuminate\Support\Carbon $deleted_at
+ */
+
+ class CandidateSearchRequest extends Model
```

Comment:
Any reason why this class is not named `PoolCandidateSearchRequest`?

Reply:
Uhh no there's no good reason. I'll change it.